crates/weaver_live_check/README.md

```diff
@@ -0,0 +1,239 @@
+        templates["Jinja Templates"]
+        output["Output Format"]
+    end
+
+    file --> ingesters
+    stdin --> ingesters
+    otlp --> ingesters
+
+    registry -- "Loaded for comparison" --> advisors
+    ingesters -- "Intermediary representations" --> advisors
+
+    builtin --> advice
+    external --> advice
+
+    advice -- "Transformed via" --> templates
+    templates --> output
+```
+
+## Ingesters
+
+Sample data can have various levels of detail; from a simple list of attribute names, to a full OTLP signal structure. This data can come from different sources: files, stdin, OTLP. Therefore you need to choose the appropriate `Ingester` for your job:
+
+- `attribute_file | AF | af` => for attribute name or name=value pair text files
+- `attribute_stdin | AS | as` => for attribute name or name=value pairs on stdin
+- `attribute_json_file | AJF | ajf` => for an array of attribute samples in a json file
+- `attribute_json_stdin | AJS | ajs` => for an array of attribute samples in json on stdin
+- `attribute_otlp | AO | ao` => for attributes extracted from OTLP signals
+
+Some `Ingesters`, like `attribute_stdin` and `attribute_otlp`, can stream the input data so you receive output at the command line as it comes in. This is really useful in live debugging sessions allowing you to breakpoint, step through your code and see live assessment as the data is received in Weaver.
+
+## Advisors
+
+Sample entities are assessed by the set of `Advisors` to produce a list of `Advice` for each one. Built-ins check for fundamental compliance with the `Registry` supplied, for example `missing_attribute` and `type_mismatch`.
+
+Beyond the fundamentals, external `Advisors` can be defined in Rego policies. The OpenTelemetry Semantic Conventions rules are included out-of-the-box by default. They provide `Advice` on name-spacing and formatting aligned with the standard. These default policies can be overridden at the command line with your own.
+
+### Advice
+
+As mentioned, a list of `Advice` is returned in the report for each sample entity. The snippet below shows `Advice` from two `Advisors`. A builtin is providing `missing_attribute` and a default Otel Rego policy is providing `extends_namespace`. The fields of `Advice` are intended to be used like so:
+
+- `advisory`: _string_ - one of `violation`, `improvement` or `information` with that order of precedence. Weaver will return with a non-zero exit-code if there is any `violation` in the report.
+- `key`: _string_ - a simple machine readable string key to represent the advice type
+- `message`: _string_ - a verbose string describing the advice
+- `value`: _any_ - a pertinent entity associated with the advice
+
+```json
+{
+  "all_advice": [
+    {
+      "advisory": "violation",
+      "key": "missing_attribute",
+      "message": "Does not exist in the registry",
+      "value": "aws.s3.extension.name"
+    },
+    {
+      "advisory": "information",
+      "key": "extends_namespace",
```

Comment:
This policy probably needs a link to explain *why* here. 

Right now, if I were unfamiliar with OTel's naming rules, this wouldn't make a lot of sense.

Reply:
Removed from readme. It was just an example.